<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datastoragescreen</class>
 <widget class="QWidget" name="datastoragescreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backgroundButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="viewSystemLogsButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View System Logs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewHealthLogsButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View Health Logs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPageButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousPageButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentPageLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Page: X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="entry1Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Entry 1: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="entry2Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Entry 2: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="entry3Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Entry 3: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="entry4Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Entry 4: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="entry5Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Entry 5: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentViewLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Currently Viewing: System Logs</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>461</width>
     <height>361</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>459</width>
      <height>359</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="scrollPageMainLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Page: X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
